src/main/scala/firrtl/options/StageAnnotations.scala

```diff
@@ -14,6 +17,23 @@ sealed trait StageOption { this: Annotation => }
   */
 trait Unserializable { this: Annotation => }
 
+trait HowToSerialize { this: Annotation =>
+
+  protected def baseFileName: String
+
+  protected def suffix: Option[String]
+
+  def howToSerialize: Option[Stream[Byte]]
+
+  def howToResume(file: File): Option[AnnotationSeq]
```

Comment:
also missing scaladoc

Personally I have no idea what `howToResume` is supposed to do

Reply:
Scaladoc added. Names not yet changed.